redash/models/base.py

```diff
@@ -42,6 +45,21 @@ def apply_pool_defaults(self, app, options):
 make_searchable(db.metadata, options={"regconfig": "pg_catalog.simple"})
 
 
+# IAM database authentication for AWS RDS
+# See https://docs.aws.amazon.com/AmazonRDS/latest/UserGuide/UsingWithRDS.IAMDBAuth.html
```

Comment:
For me, it was not so easy to find the code is the best way or not without searching the internet.
I think, it is better to add a bit more reference like below so that people know that this is the way SQLAlchemy is recommending.

  Generating dynamic authentication tokens - 
  https://docs.sqlalchemy.org/en/20/core/engines.html#generating-dynamic-authentication-tokens